### PR TITLE
Update rewards-troubleshooting.mdx

### PR DESCRIPTION
**Pull request by ipanagiv**

### CODE REVIEW

File: docs/rewards/rewards-troubleshooting.mdx [hunk 1]
```diff
@@ -7,20 +7,125 @@ slug: /project/rewards-troubleshooting
...
+That either means that your station was offline or not connected to the network. 
 
-### Location Not Verified
+Here's what you need to do:
 
+### For the WS-1000 (WiFi/M5 version)
 
-### User Relocation Penalty
\ No newline at end of file
+1. Check if the station's battery level is low through mobile app settings.
```
Comment (PavlosTze) [inline docs/rewards/rewards-troubleshooting.mdx:30]: As I suggested in the Linear task, before starting that list, we need to include a screenshot like this one [here](https://docs.weatherxm.com/img/wxm-devices/wifi-m5-bundle/m5-ui-new.png), showing the M5 and explaining what each icon at the top right corner means.

File: docs/rewards/rewards-troubleshooting.mdx [hunk 2]
```diff
@@ -7,20 +7,125 @@ slug: /project/rewards-troubleshooting
...
+
+Weather data packets failed to pass our quality checks. We evaluate all data packets sent by our weather stations using our QoD (Quality of Data) algorithm. You can read more about this algorithm [here](https://docs.weatherxm.com/project/quality-of-data).
+There are several reasons that you may experience this error, which includes:
+
+- Out of bounds data
+- Spikes on data 
+- Constant values
+- Frozen sensor
+
+Check out our [Deployment Examples](/deployment-examples) and [Best Practices](/wxm-devices/wifi-m5-bundle/install-weather-station#best-practices-for-proper-station-installation) to ensure that your station is deployed correctly,
```
Comment (PavlosTze) [inline docs/rewards/rewards-troubleshooting.mdx:53]: We need a sentence somewhere here prompting the user to defrost the sensor/station if it's frozen.

File: docs/rewards/rewards-troubleshooting.mdx [hunk 3]
```diff
@@ -7,20 +7,125 @@ slug: /project/rewards-troubleshooting
...
+
+## Weather Data Gaps
+Some data gaps were detected during our data quality checks.
+Indoor or outdoor units are not properly connected to the network, or to each other.
+
+Here's what you need to do:
+
+### For the WS-1000 (WiFi/M5 version)
+
+1. Check if the station's battery level is low through mobile app settings.
```
Comment (PavlosTze) [inline docs/rewards/rewards-troubleshooting.mdx:65]: As I suggested in the Linear task, before starting that list, we need to include a screenshot like this one [here](https://docs.weatherxm.com/img/wxm-devices/wifi-m5-bundle/m5-ui-new.png), showing the M5 and explaining what each icon at the top right corner means.

**Review by PavlosTze — CHANGES_REQUESTED** [review]

File: docs/rewards/rewards-troubleshooting.mdx (hunk 3 above)
Comment (ipanagiv) [reply docs/rewards/rewards-troubleshooting.mdx:65]: I have seen what you have suggested, I don't believe it is a dealbreaker.

File: docs/rewards/rewards-troubleshooting.mdx (hunk 1 above)
Comment (PavlosTze) [reply docs/rewards/rewards-troubleshooting.mdx:30]: We definitely need this to be added, if not now, at least until the end of the week. I have seen numerous messages on Discord from users not understanding what each icon means and we have nowhere that piece of information in our docs.

**Review by PavlosTze — APPROVED** [review]

**Pull request merged**